script/Ciel/Day 9/09_01_CIEL09_1C.txt

```diff
@@ -173,75 +193,88 @@ Ah, but if I just carelessly love dive you, then Matriarch-chan will kill me! Ky
 [sha:d8becfe7ca5e0bd4f2ee0c8ebe8156511bfc67f7]{
 -- Page 66, Offset 22304.
 -- ……遠野邸にいないと思ったら、こんな仕事していたんだこの人……というか、本当に医者だったんだ。
-I knew she wasn't at the Tohno mansion, but to think that she would be doing something like this... wait, this means she's actually a real doctor.
+I knew she wasn't at the Tohno mansion today, but I'd never have imagined she'd be here instead...
+ Wait, does this mean she's actually a real doctor?
 }
 [sha:5b78affb0f3aed4000c21506dc7206449c1fe2fd]{
 -- Page 67, Offset 22305.
 -- 「なーんてウソウソ、はじめから知ってたっていうの！
-"Just kidding. I knew from the start!
+"Juuust kidding. I already knew I'd find you here!
 }
 [sha:51209864f34337880f4e3ef6a3c929408ca43e1c]{
 -- Page 67, Offset 22306.
 -- 学校でお医者さんプレイとかアタシも初めてだし？　ハイトク的っていうの？　なんにせよ初物はサイコウよねぇ。ここなら当主チャンの目も届かないし！
-It's also my first time playing doctor at a school, you know? It's kind of like, immoral? Anyway, I like them best when they're young and fresh! Besides, Matriarch-chan won't find out about what happens here!
+This is actually my first time playing doctor at a school, you know?
+ It feels kinda... immoral?
+ I just can't resist them when they're young and fresh!
+ And besides, it's not like Matriarch-chan will find out about anything that happens here!
 }
 [sha:478300bb9da5a49017f986c320e1bbbc63109795]{
 -- Page 67, Offset 22307.
 -- では出席番号ジュウサン番、遠野志貴君。カンネンして座りたまえ。逃げたら内申書にバツつけるゾ♡」
-Now then, seat number 33, Tohno Shiki-kun. Cease all resistance and be seated! If you try to run away, I'll put a bad mark on your school record♡"
+Now then, seat number 33, Tohno Shiki-kun. Cease all resistance and be seated!
+ If you try to run away, I'll put a bad mark on your school record ♡"
 }
 [sha:6a67488c9819966510c68fe966feacef472d3e11]{
 -- Page 68, Offset 22308.
 -- 阿良句氏は注射の準備などをしつつ、頭のゆだった発言をする。
 // Really unusual expression, that 頭のゆだった. I imagine it's like staying in the sun too long and being kinda delirious, unable to think straight.
-Ms. Arach says feverishly while preparing to administer the injection.
+Ms. Arach keeps on babbling feverishly as she busies herself preparing to administer the injection.
 }
 [sha:2ea660a5e1f70c559b59aaf46208ebbd37bc939e]{
 -- Page 68, Offset 22309. Mods: @n.
 -- いちいち反応すると阿良句氏を喜ばせるだけなので、黙って椅子に座って右腕を差し出した。
-Since reacting even the slightest bit would serve to please her, I silently sit on the chair and hold my right arm out.
+// Is he left handed?
```

Comment:
he holds his knife in his right hand in every pic i can find. I think he might just like to _live dangerously_
![image](https://user-images.githubusercontent.com/16092757/197910948-f747c0a0-4a69-4a26-9292-d151a086d912.png)